<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_recommendation">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>추천</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_keyword">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>641</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>What should I wear today</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_recommendation">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_recommendation_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>211</width>
     <height>191</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhTime|Qt::ImhUppercaseOnly</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_recommendation_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_recommendation_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>600</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_recommendation_5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>370</y>
     <width>211</width>
     <height>191</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhTime|Qt::ImhUppercaseOnly</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_recommendation_6">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>560</y>
     <width>211</width>
     <height>191</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhTime|Qt::ImhUppercaseOnly</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_keyword_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_keyword_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_keyword_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>680</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_recommendation_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>추천</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_recommendation_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>530</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>추천</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_recommendation_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>720</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>추천</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>740</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>471</width>
     <height>791</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/다운로드 (1).jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>461</width>
     <height>791</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/다운로드.jpg</pixmap>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>btn_recommendation</zorder>
  <zorder>le_keyword</zorder>
  <zorder>graphicsView</zorder>
  <zorder>label</zorder>
  <zorder>lbl_recommendation</zorder>
  <zorder>lbl_recommendation_2</zorder>
  <zorder>graphicsView_2</zorder>
  <zorder>lbl_recommendation_3</zorder>
  <zorder>graphicsView_3</zorder>
  <zorder>lbl_recommendation_4</zorder>
  <zorder>lbl_recommendation_5</zorder>
  <zorder>lbl_recommendation_6</zorder>
  <zorder>le_keyword_2</zorder>
  <zorder>le_keyword_3</zorder>
  <zorder>le_keyword_4</zorder>
  <zorder>btn_recommendation_2</zorder>
  <zorder>btn_recommendation_3</zorder>
  <zorder>btn_recommendation_4</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
